app/views/service/_svcs_show.html.haml

```diff
@@ -38,8 +40,10 @@
             .form-horizontal.static
               .form-group
                 .col-md-9
-                  - htm = job.raw_stdout('html').gsub('"', '\"').gsub("'", "\\\\'")
-                  %div{'ng-bind-html'=>"vm.sanitizeRenderHtml('#{htm}')"}
+                  - htm = @job.raw_stdout('html')
+                  %div#std_output2
+                    %miq-sanitize{:value => htm}
 :javascript
   miq_tabs_init('#services_tab');
-  miq_bootstrap('#service_details', 'sanitizeRender');
+  miq_bootstrap('#std_output', 'miq.helpers');
```

Comment:
@syncrou i am tying the div that holds standard output to miq_bootstrap object, previously we were attaching the whole outer div but in this case i only need the specific to be attached. `miq.helpers` includes ngSanitize directive that is being used.